src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Support for Resource attributes in OpenTelemetry.Exporter.InfluxDB, allowing
+resource attributes to be passed as InfluxDB tags.
```

Comment:
nit:
```suggestion
  resource attributes to be passed as InfluxDB tags.
```

Reply:
Hmm will this still be in line with the rest of the file?